jenkins_pipelines/environments/salt-shaker/manager-salt-shaker-saltstack-leap154

```diff
@@ -4,7 +4,18 @@ node('salt-shaker-tests') {
     properties([
         buildDiscarder(logRotator(numToKeepStr: '20', daysToKeepStr: '4', artifactNumToKeepStr: '3')),
         disableConcurrentBuilds(),
-        pipelineTriggers([cron('H 0 * * *')]),
+        pipelineTriggers([
+            URLTrigger(
+                cronTabSpec: '* * * * *',
+                triggerLabel: "salt-shaker-tests",
+                labelRestriction: true,
+                entries: [URLTriggerEntry(
+                    url: 'https://download.opensuse.org/repositories/systemsmanagement:/saltstack/15.4/repodata/repomd.xml',
+                    contentTypes: [MD5Sum()]
+                )]
+            ),
+            cron('H 0 * * *')],
+        ),
```

Comment:
In order to save HW resources, we decided to add the trigger at repository change only for some OSes, not for all of them.

In this sense, the idea is to only trigger by repository change on the latest version of each OSes, the rest only at night.